tests/test_tasks/test_indexing.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2025 Robotics and AI Institute LLC. All rights reserved.
```

Comment:
extreme nit: whitespace after this header. in fact, i thought that the linter enforced this already, i'm kinda surprised to see this